scripts/analysis.py

```diff
@@ -56,188 +25,85 @@ def get_agent_ids(cursor, facility):
     cursor: cursor
         cursor for sqlite3
     facility: str
-        name of facility type
+        name of facility type as described in spec
 
     Returns
     -------
-
     sink_id: list
-        list of all the sink agentId values.
+        list of all the sink agentId values as string.
     """
+    agents = cursor.execute("SELECT * FROM agententry WHERE spec LIKE '%" +
+                            facility + "%' COLLATE NOCASE").fetchall()
 
-    cur = cursor
-    agent_id = []
-    agent = cur.execute("select * from agententry where spec like '%"
-                        + facility + "%'").fetchall()
+    return list(str(agent[1]) for agent in agents)
```

Comment:
Due to sqlite queries, getting a list of agentid strings turned out to be more useful than strings.
The main reason for this is elaborated in the comment above.